op-challenger/fault/responder.go

```diff
@@ -79,18 +89,25 @@ func (r *faultResponder) BuildTx(ctx context.Context, response types.Claim) ([]b
 	}
 }
 
-// CanResolve determines if the resolve function on the fault dispute game contract
-// would succeed. Returns true if the game can be resolved, otherwise false.
-func (r *faultResponder) CanResolve(ctx context.Context) bool {
+// CallResolve determines if the resolve function on the fault dispute game contract
+// would succeed. Returns the game status if the call would succeed, errors otherwise.
+func (r *faultResponder) CallResolve(ctx context.Context) (uint8, error) {
```

Comment:
Can we return `GameStatus` here instead of `uint8`? That way the details of the contract representation are better encapsulated and we get more specify typing for callers.